web/src/components/forms/create-template-form.tsx

```diff
@@ -26,33 +28,43 @@ export function CreateTemplateForm({
   specifiedTeam = undefined,
 }: CreateTemplateFormProps) {
   const {user} = useAuth();
-  const QueryClient = useQueryClient();
+  const queryClient = useQueryClient();
   const {data: teams} = useGetTeams(user);
 
   // can they create projects outside team?
-  const canCreateGlobally = useIsAuthorisedTo({action: Action.CREATE_TEMPLATE});
+  const canCreateGlobally = useIsAuthorisedTo({
+    action: Action.CREATE_TEMPLATE,
+  });
 
   const fields: Field[] = [
     {
       name: 'name',
-      label: 'Name',
+      label: 'Template Name',
+      description:
+        'Pick something clear—you can always rename it later in the designer.',
```

Comment:
This is a weird description, and you can't rename it in designer - you now need to use edit - maybe "A short display name for the new template" would suffice